topdown/graphql.go

```diff
@@ -5,32 +5,19 @@
 package topdown
 
 import (
+	"encoding/json"
 	"fmt"
 	"strings"
 
-	gqltop "github.com/vektah/gqlparser/v2"
 	gqlast "github.com/vektah/gqlparser/v2/ast"
 	gqlparser "github.com/vektah/gqlparser/v2/parser"
 	gqlvalidator "github.com/vektah/gqlparser/v2/validator"
+	_ "github.com/vektah/gqlparser/v2/validator/rules" // Note(philipc): Registers rule handlers for the validator.
```

Comment:
The `gqlparser` library uses side-effecting imports to load its GraphQL validation ruleset. This is annoying, but necessary, because without it, none of the validation functions we care about (e.g. verifying a query respects a given schema) work correctly.